packages/sanity/src/components/previews/portableText/BlockImagePreview.tsx

```diff
@@ -86,7 +88,7 @@ export function BlockImagePreview(props: BlockImagePreviewProps) {
       {description && (
         <Box paddingX={2} paddingY={3}>
           <Text muted size={1}>
-            {renderPreviewNode(description, 'block')}
+            {renderPreviewNode(description as any, 'block')}
```

Comment:
Same here, typed as ReactNode | ComponentType, is that correct?

Reply:
Casting it to `any` fixes this error:
```bash
packages/sanity/src/components/previews/portableText/BlockImagePreview.tsx:91:32 - error TS2345: Argument of type 'string | number | true | ReactElement<any, string | JSXElementConstructor<any>> | ReactFragment | ReactPortal | ComponentClass<...> | FunctionComponent<...>' is not assignable to parameter of type 'ReactNode | ComponentType<{ layout: "block" | "blockImage"; }>'.
  Type 'ComponentClass<{ layout: "blockImage"; }, any>' is not assignable to type 'ReactNode | ComponentType<{ layout: "block" | "blockImage"; }>'.
    Type 'ComponentClass<{ layout: "blockImage"; }, any>' is not assignable to type 'ComponentClass<{ layout: "block" | "blockImage"; }, any>'.
      Types of property 'getDerivedStateFromProps' are incompatible.
        Type 'GetDerivedStateFromProps<{ layout: "blockImage"; }, any> | undefined' is not assignable to type 'GetDerivedStateFromProps<{ layout: "block" | "blockImage"; }, any> | undefined'.
          Type 'GetDerivedStateFromProps<{ layout: "blockImage"; }, any>' is not assignable to type 'GetDerivedStateFromProps<{ layout: "block" | "blockImage"; }, any>'.
            Types of parameters 'nextProps' and 'nextProps' are incompatible.
              Type 'Readonly<{ layout: "block" | "blockImage"; }>' is not assignable to type 'Readonly<{ layout: "blockImage"; }>'.
                Types of property 'layout' are incompatible.
                  Type '"block" | "blockImage"' is not assignable to type '"blockImage"'.
                    Type '"block"' is not assignable to type '"blockImage"'.

91             {renderPreviewNode(description, 'block')}
```